src/types.deno.ts

```diff
@@ -114,7 +114,7 @@ export class InputFile {
                     "Reading files by path requires a Deno environment",
                 );
             }
-            using file = await Deno.open(data);
+            const file = await Deno.open(data);
             return file.readable;
```

Comment:
Resource Leak, when I have time, I will try to fix it. (My idea is to use `AsyncIterator` along with `[Symbol.dispose]`)